charts/prometheus-druid-exporter/Chart.yaml

```
@@ -1,16 +1,18 @@
 ---
 apiVersion: v1
-appVersion: "v0.8.0"
+appVersion: "v0.11"
 description: Druid exporter to monitor druid metrics with Prometheus
 engine: gotpl
 name: prometheus-druid-exporter
-version: 0.11.0
+version: 0.11.1
```

new feature should raise minor version

---

i did already, recheck the committed code please

---

No, you changed the patch version. See: https://semver.org/

```suggestion
version: 0.12.0
```